skatepark/inc/patterns/pre-footer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Pre Footer.
+ *
+ * @package Skatepark
+ */
+
+return array(
+	'title'      => __( 'Footer', 'skatepark' ),
+	'categories' => array( 'skatepark' ),
+	'content'    => '<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"60px","bottom":"60px"}}},"className":"pre-footer"} -->
+	<div class="wp-block-group alignwide pre-footer" style="padding-top:60px;padding-bottom:60px"><!-- wp:columns -->
+		<div class="wp-block-columns"><!-- wp:column -->
+		<div class="wp-block-column"><!-- wp:site-title {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"large"} /-->
+		
+		<!-- wp:paragraph {"fontSize":"small"} -->
+		<p class="has-small-font-size">' . esc_html__( 'Skate ipsum dolor sit amet, poseur nollie casper pop shove-it. Kickturn noseblunt tailslide.', 'skatepark' ) . '</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:social-links {"iconColor":"primary","iconColorValue":"#000000","className":"is-style-logos-only"} -->
```

Comment:
I thought this would break validation but it works! can't be achieved through the editor but it works from the pattern and it makes a lot of sense, thanks for pointing it out!